sync/rpc/server.go

```diff
@@ -209,9 +228,11 @@ func (s *SyncAgentServer) BackupCreate(ctx context.Context, req *BackupCreateReq
 	reply := &BackupCreateReply{
 		Backup: backupID,
 	}
-	fmt.Fprintf(os.Stdout, reply.Backup)
 
-	s.backupList.Store(backupID, replicaObj)
+	s.backup.BackupCreate(backupID, replicaObj)
+	if err := s.backup.BackupUpdate(); err != nil {
```

Comment:
Caller shouldn't need to remember calling `BackupUpdate` after BackupCreate.  It should be done automatically since there is no situation that you won't call `BackupUpdate` after BackupCreate.

Also, call it `backuplist.BackupAdd()`, since you're not creating a new object. You just pass some information in.

And call BackupUpdate() `Refresh()` since you're not updating the object. It's updating the list.

And the `Refresh()` function shouldn't return any error because it's all in memory. Unless there is a bug, there shouldn't be any error.